src/screens/Contact/Contact.js

```diff
@@ -446,30 +510,58 @@ class Contact extends React.Component<Props, State> {
-            />
-            {disableSend &&
-            <Wrapper regularPadding style={{ marginTop: 30, alignItems: 'center' }}>
-              <MessageTitle>{ sendingBlockedMessage.title }</MessageTitle>
-              <Message>{ sendingBlockedMessage.message }</Message>
-              {smartWalletStatus.status === SMART_WALLET_UPGRADE_STATUSES.ACCOUNT_CREATED &&
-              <Button
-                marginTop="20px"
-                height={52}
-                title="Deploy Smart Wallet"
-                disabled={smartWalletStatus.status === SMART_WALLET_UPGRADE_STATUSES.DEPLOYING}
-                onPress={() => deploySmartWallet()}
-              />
-              }
-            </Wrapper>
-            }
-          </CircleButtonsWrapper>
-         }
+            <ContentWrapper>
+              <CircleButtonsWrapper>
+                <CircleButton
+                  disabled={disableSend}
+                  label="Send"
+                  icon={iconSend}
+                  onPress={() => this.onSendPress(displayContact)}
+                />
+                {disableSend &&
+                <Wrapper regularPadding style={{ marginTop: 30, alignItems: 'center' }}>
+                  <MessageTitle>{ sendingBlockedMessage.title }</MessageTitle>
+                  <Message>{ sendingBlockedMessage.message }</Message>
+                  {smartWalletStatus.status === SMART_WALLET_UPGRADE_STATUSES.ACCOUNT_CREATED &&
+                  <Button
+                    marginTop="20px"
+                    height={52}
+                    title="Deploy Smart Wallet"
+                    disabled={smartWalletStatus.status === SMART_WALLET_UPGRADE_STATUSES.DEPLOYING}
+                    onPress={() => deploySmartWallet()}
+                  />
+                  }
+                </Wrapper>
+                }
+              </CircleButtonsWrapper>
+              { /* <CollapsibleSection
+                label="game of badges."
+                collapseContent={
+                  <FlatList
+                    data={contactBadges}
+                    keyExtractor={(item) => item.id.toString()}
+                    renderItem={this.renderBadge}
+                    style={{ width: '100%' }}
+                    contentContainerStyle={[
+                      { paddingHorizontal: 10 },
+                      !contactBadges.length ? { width: '100%', justifyContent: 'center' } : {},
+                      ]}
+                    horizontal
+                    initialNumToRender={5}
+                    removeClippedSubviews
+                    ListEmptyComponent={this.renderEmptyBadgesState}
+                  />
+                }
+                onPress={this.toggleBadgesSection}
+                open={isBadgesSectionOpen}
+                onAnimationEnd={
+                  isBadgesSectionOpen
+                  ? () => { this.scroll.scrollToEnd(); }
+                  : () => {}
+                }
+              /> */}
```

Comment:
Do we want to keep this? There are some other commented blocks that I'd remove if they are not required.

Reply:
I'd leave it - it's, as I've wrote in the description, is commented out till we allow users to manage badge visibility